src/atomate2/vasp/sets/BaseMPR2SCANRelaxSet.yaml

```diff
@@ -0,0 +1,160 @@
+# Default VASP settings for calculations in the Materials Project
+# using the Regularized-Restored Strongly Constrained and Appropriately
+# Normed functional (r2SCAN).
+INCAR:
+  ALGO: ALL
+  EDIFF: 1.e-05
+  EDIFFG: -0.02
+  ENAUG: 1360
+  ENCUT: 680
+  IBRION: 2
+  ISIF: 3
+  ISMEAR: 2
+  ISPIN: 2
+  KSPACING: 0.22
+  LAECHG: True # Output AECCAR
+  LASPH: True
+  LCHARG: True # Output CHGCAR
+  LELF: True # Output ELFCAR
+  LMIXTAU: True
+  LORBIT: 11
+  LREAL: Auto
+  LVTOT: True # Output LOCPOT
+  LWAVE: False # Don't output WAVECAR
+  METAGGA: R2SCAN
+  NELM: 200
+  NSW: 99
+  PREC: Accurate
+  SIGMA: 0.2
+  MAGMOM:
+    Ce: 5
+    Ce3+: 1
+    Co: 0.6
+    Co3+: 0.6
+    Co4+: 1
+    Cr: 5
+    Dy3+: 5
+    Er3+: 3
+    Eu: 10
+    Eu2+: 7
+    Eu3+: 6
+    Fe: 5
+    Gd3+: 7
+    Ho3+: 4
+    La3+: 0.6
+    Lu3+: 0.6
+    Mn: 5
+    Mn3+: 4
+    Mn4+: 3
+    Mo: 5
+    Nd3+: 3
+    Ni: 5
+    Pm3+: 4
+    Pr3+: 2
```

Comment:
I note that `MPSCANRelaxSet.yaml` in pymatgen does not include any MAGMOMs. I think it's good to have them here; just flagging as a potential source of discrepancies in test cases, etc.

Reply:
Yes, this was a deliberate choice. They are identical with the MAGMOMs in [BaseMPGGASet.yaml](https://github.com/materialsproject/atomate2/blob/c98cf7c8744dbe001c1ffedf74430242db3015d9/src/atomate2/vasp/sets/BaseMPGGASet.yaml). Btw, we thought about extracting them into a separate `MAGMOMs.yaml` file and allowing for parent-child inheritance similar to `pymatgen`'s YAML sets but felt a bit clunky and over-engineered. Happy to discuss more though.

Reply:
Note that we also added a default KSPACING, SIGMA, and ISMEAR to the input set that way if someone tried to use the MP settings on some random Atomate2 VASP workflow, they aren't running without any k-points. It also made writing the PBEsol step easier.